shared/pkg/utils/image_helper/image_helper.go

```diff
@@ -225,14 +225,14 @@ func GetImageLayerCommands(imageName string, sharedConf *sharedconfig.Config) ([
 	return layerCommands, nil
 }
 
-func GetHashFromRepoOrManifestDigest(repoDigests []string, manifestDigest string, imageName string) string {
+func GetHashFromRepoDigestsOrImageID(repoDigests []string, imageID string, imageName string) string {
 	hash := GetHashFromRepoDigest(repoDigests, imageName)
 	if hash == "" {
-		// set hash using ManifestDigest if RepoDigest is missing
-		if idx := strings.Index(manifestDigest, ":"); idx != -1 {
-			hash = manifestDigest[idx+1:]
+		// set hash using ImageID (https://github.com/opencontainers/image-spec/blob/main/config.md#imageid) if repo digests are missing
+		if idx := strings.Index(imageID, ":"); idx != -1 {
+			hash = imageID[idx+1:]
 		} else {
-			hash = manifestDigest
+			hash = imageID
 		}
```

Comment:
I think we could use strings.Cut here:
```
_, remainder, found := strings.Cut(imageID, ":")
if found {
   hash = remainder
}
```

Reply:
I agree, In the case of the `imageID` doesn't contain `:` we need to add
```
} else {
    hash = imageID
}
```

Reply:
done [`9b55c2c` (#510)](https://github.com/openclarity/kubeclarity/pull/510/commits/9b55c2cfbd846f4886b55b16e8fd8d0c9e9d4262)